ajax/libs/urljs/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "urljs",
+  "version": "1.1.0",
+  "description": "A lightweight JavaScript library to manipulate the page url.",
+  "main": "url.min.js",
+  "filename": "url.min.js",
+  "directories": {
+    "example": "example"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jillix/url.js.git"
+  },
+  "keywords": [
+    "url",
+    "manipulate",
+    "browser"
+  ],
+  "author": "jillix <contact@jillix.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jillix/url.js/issues"
+  },
+  "homepage": "https://github.com/jillix/url.js",
+  "npmName": "urljs",
```

Comment:
What about if I don't provide that? Shouldn't it update automatically using `name` field?